controls/radautocompletebox/styles-and-templates/customizing-drop-down-itemtemplate.md

```diff
@@ -1,18 +1,16 @@
 ---
-title: Customizing the DropDownItemTemplate
-page_title: Customizing the DropDownItemTemplate
-description: Customizing the DropDownItemTemplate
+title: Customizing the DropDownItemTemplate and DropDownItemTemplateSelector
+page_title: Customizing the DropDownItemTemplate and DropDownItemTemplateSelector
+description: The RadAutoCompleteBox control expose DropDownItemTemplate and DropDownItemTemplateSelector which can be used to customize the drop down item.
 slug: radautocompletebox-customizing-drop-down-itemtemplate
 tags: customizing,the,dropdownitemtemplate
 published: True
 position: 0
 ---
 
-# Customizing the DropDownItemTemplate
+# Customizing the DropDownItemTemplate and DropDownItemTemplateSelector
 
-In the __RadAutoCompleteBox__ control you can customize the way items are rendered in the DropDown portion of the control.
-
-By customizing the __DropDownItemTemplate__ of the control you can easily show different template for the items in the DropDown portion of the control.
+In the __RadAutoCompleteBox__ control you can customize the way items are rendered in the DropDown portion of the control. By customizing the __DropDownItemTemplate__ or __DropDownItemTemplateSelector__ of the control you can easily show different template for the items in the DropDown portion of the control.

```

Comment:
We can rephrase this a little better. For example: 
In the __RadAutoCompleteBox__ control you can customize the way items are rendered in the DropDown portion of the control. You can do that by setting the __DropDownItemTemplate__ or __DropDownItemTemplateSelector__ properties. They allow you to show different templates for the DropDown items.